src/main/java/io/cryostat/net/web/http/api/v2/ApiListingHandler.java

```diff
@@ -35,31 +35,42 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-package io.cryostat.net.web.http.generic;
+package io.cryostat.net.web.http.api.v2;
 
-import java.util.Comparator;
-import java.util.Map;
+import java.net.URI;
+import java.net.URL;
+import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 import javax.inject.Inject;
 
+import io.cryostat.net.AuthManager;
+import io.cryostat.net.web.WebServer;
+import io.cryostat.net.web.http.HttpMimeType;
 import io.cryostat.net.web.http.RequestHandler;
 import io.cryostat.net.web.http.api.ApiVersion;
 
 import com.google.gson.Gson;
+import com.google.gson.annotations.SerializedName;
 import dagger.Lazy;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import io.vertx.core.http.HttpMethod;
-import io.vertx.ext.web.RoutingContext;
 
-class ApiListingHandler implements RequestHandler {
+class ApiListingHandler extends AbstractV2RequestHandler<ApiListingHandler.ApiResponse> {
 
+    private final Lazy<WebServer> webServer;
     private final Lazy<Set<RequestHandler>> handlers;
     private final Gson gson;
```

Comment:
Is this variable used anywhere?

Reply:
Sorry I meant to highlight `private final Gson gson`

Reply:
Ah, good catch. Not anymore now that I moved this over to extending the V2 base handler abstract class, so this implementation doesn't do serialization directly on its own anymore.

Reply:
After extending the V2 base handler, does that mean `apiGetHandler`'s `apiVersion` is V2 instead of `generic`?

Reply:
> Ah, good catch. Not anymore now that I moved this over to extending the V2 base handler abstract class, so this implementation doesn't do serialization directly on its own anymore.

After extending the V2 base handler, does that mean `apiGetHandler`'s `apiVersion` is `V2` instead of `generic`?